app/models/authorized_keys_file.rb

```diff
@@ -2,23 +2,19 @@
 
 # Class to rewrite the authorized_keys file
 class AuthorizedKeysFile
+  LOCK_FILE = 'authorized_keys.lock'
+
   class << self
     def write
       keys = PublicKey.all.map do |key|
         "#{key.key.strip} #{key.name}"
       end
 
-      Filelock(lock_file, timeout: 5) do
+      FileLockHelper.exclusively(LOCK_FILE, timeout: 5) do
```

Comment:
Can you change the `5` to `5.seconds`, just to be more explicit about it?

Reply:
Done in 6ec95ce7